src/components/TabsView/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react"
+import Tabs from "@mui/material/Tabs"
+import Tab from "@mui/material/Tab"
+import Typography from "@mui/material/Typography"
+import Box from "@mui/material/Box"
+import { TabsWrapper } from "./styles"
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  dir?: string
+  index: number
+  value: number
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`full-width-tabpanel-${index}`}
+      aria-labelledby={`full-width-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+export default function TabsView({ pdfList }: any) {
+  const [value, setValue] = React.useState(0)
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue)
+  }
+
+  return (
+    <Box sx={{ bgcolor: "background.paper", width: 500 }}>
+      <TabsWrapper position="static">
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          indicatorColor="primary"
+          textColor="inherit"
+          variant="fullWidth"
+          aria-label="full width tabs example"
+        >
+          {pdfList.map((pdf: any, index: number) => (
+            <Tab
+              style={
+                value === index
+                  ? { backgroundColor: "#fff", color: "#000" }
+                  : {}
+              }
+              label={pdf.title}
+              key={pdf.title}
+            />
+          ))}
+        </Tabs>
+      </TabsWrapper>
+      {pdfList.map((pdf: any) => (
+        <TabPanel value={value + 1} index={pdf.id} key={pdf.title}>
+          {pdf.id}
```

Comment:
You can use the index here. I don't think the pdf list will come with an id

Reply:
this is just to illustrate the change of tab, it shouldn't show up on the final page